releases/release-5.3.0.md

```diff
@@ -0,0 +1,388 @@
+
+    > **Note:**
+    >
+    > When the number of clients is small and the PD leader CPU is not full, it is NOT recommended to enable PD Follower Proxy.
+
+    - Support setting the maximum waiting time through the system variable [`tidb_tso_client_batch_max_wait_time`](/system-variables.md#tidb_tso_client_batch_max_wait_time-new-in-v53). This time is the waiting time for batching TSO requests when PD client requests TSO in batches. The unit of this time is milliseconds. In case that PD has a high TSO requests load, you can reduce the load and improve the throughput by increasing the waiting time to get a larger batch size.
+
+    > **Note:**
+    >
+    > When the TSO request load is not high, it is NOT recommended to modify this variable.
+
+    [User document](/system-variables.md#tidb_tso_client_batch_max_wait_time-new-in-v53),[#3149](https://github.com/tikv/pd/issues/3149)
+
+### Stability
+
+- **Support Online Unsafe Recovery after stores permanently damaged (experimental feature)**
+
+    Support the `unsafe remove-failed-stores` command that allows data unsafe recovery. Suppose that multiple replicas of data encounter problems like permanent damage (such as disk damage), and this issue causes the data in an application to be not readable or writable. In this case, Online Unsafe Recovery using PD can recover the data to be readable or writable again.
+
+> **Warning:**
+>
+> - This feature is a lossy recovery, so TiKV cannot guarantee data integrity and data indexes integrity after using the feature.
+> - Online unsafe recovery is an experimental feature, and it is **not** recommended to use it in the production environment. The interface, strategy, and internal implementation of the feature might be changed when it is finally released. Also, although the feature is tested in some scenarios, it is not thoroughly validated, so the feature might cause system unavailability.
+> - It is recommended to perform the feature-related operations with the support of the TiDB team. If some incorrect operations are performed, it might be hard to recover the cluster.
+
+[User document](/online-unsafe-recovery.md), [#10483](https://github.com/tikv/tikv/issues/10483)
+
+### Data migration
+
+- **DM replication performance enhanced**
+
+    Supports the following features to ensure lower-latency data replication from MySQL to TiDB:
+
+    - Compact multiple updates on a single row into one statement
+    - Merge batch updates of multiple rows into one statement
+
+- **Add DM OpenAPI to better maintain DM clusters (experimental feature)**
+
+    DM provides the OpenAPI feature for querying and operating the DM cluster. It is similar to the feature of [dmctl tools](./dmctl-introduction.md).
```

Comment:
@Liuxiaozhen12 Please use the https link because the referenced document **is not in this repo**.